.github/actions/docker/action.yaml

```diff
@@ -63,6 +63,9 @@ runs:
     - name: Set up Cloud SDK
       if: ${{ (steps.changed-files.outputs.any_changed == 'true') && (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.fork) }}
       uses: isarkis/setup-gcloud@40dce7857b354839efac498d3632050f568090b6 # v1.1.1
+    - name: Configure Docker auth for GCloud
+      run: |
```

Comment:
can remove the newline here and the pipe operator and just have the bare command

Reply:
Yes, but it's going to add more churn as more stuff gets inevitably added in this step. It's also somewhat more readable visually to have the shell stuff at extra indentation